### ISSUE TITLE
Help for Default buttons not responding in Flatbox Rev 5 Southpaw 0.7.9 firmware

### ISSUE THREAD

**Issue by xtang** [OpenStickCommunity/GP2040-CE#1100]
I'm using the Flatbox Rev 5 Southpaw 0.7.9 firmware, and I've found that some default buttons like "Right", "B4", and "R2" are not responding. After checking the web-config, I noticed that these buttons are assigned by an add-on by default, and I couldn't find a way to unassign them.

To troubleshoot the issue, I installed the old firmware version 0.7.5, and the problem was resolved. After reviewing the code changes, I discovered that the changes were introduced by the pull request https://github.com/OpenStickCommunity/GP2040-CE/pull/950. It seems that this is a feature, not a bug. Related codes are as follow:

```c
// Setting GPIO pins to assigned by add-on
//
#define GPIO_PIN_00 GpioAction::ASSIGNED_TO_ADDON
#define GPIO_PIN_11 GpioAction::ASSIGNED_TO_ADDON
#define GPIO_PIN_24 GpioAction::ASSIGNED_TO_ADDON
#define GPIO_PIN_25 GpioAction::ASSIGNED_TO_ADDON
#define GPIO_PIN_26 GpioAction::ASSIGNED_TO_ADDON
#define GPIO_PIN_27 GpioAction::ASSIGNED_TO_ADDON
```

Since I'm not familiar with the codebase, I believe I'm missing something. I would like to use the latest firmware version and hope that every default button can work as before. Could you please help me understand what I'm missing and provide a solution to address this issue? I'd appreciate your assistance.

**Comment by TheTrainGoes** [comment]
Hey @xtang ,

Let me take a look at this for you.

**Comment by TheTrainGoes** [comment]
Hi @xtang ,

Fixed in #1101

You can download the artifact build here to test:
https://github.com/OpenStickCommunity/GP2040-CE/actions/runs/10209056950/artifacts/1767547085

**Comment by xtang** [comment]
Thanks!

**Comment by xtang** [comment]
> Hi @xtang ,
> 
> Fixed in #1101
> 
> You can download the artifact build here to test: https://github.com/OpenStickCommunity/GP2040-CE/actions/runs/10209056950/artifacts/1767547085

It works as expected.
